Sources/AutomaticProperties.swift

```diff
@@ -107,10 +107,7 @@ class AutomaticProperties {
                 }
             }
         }
-        if let model = String(validatingUTF8: modelCode) {
```

Comment:
`warning: 'init(validatingUTF8:)' is deprecated: Use a copy of the String argument` ... not sure exactly why we were doing this or if we really needed to? @zihejia

Reply:
https://developer.apple.com/documentation/swift/string/init(validatingutf8:)-2o7g5?changes=latest_major